test/powershell/Language/Scripting/Generics.Tests.ps1

```diff
@@ -0,0 +1,74 @@
+﻿Describe "Generics support" -Tags "CI" {
+    #
```

Comment:
added

---
In reply to: [76886683](https://github.com/Powershell/Powershell/pull/2118#discussion_r76886683) [](ancestors = 76886683)